src/link.js

```diff
@@ -5,6 +5,11 @@ const { keys } = Object;
 const makeMap = (...args) => new Map(...args);
 
 export const makeEvaluateLinker = (evaluator, instanceCache = new Map()) => {
+  const tolerantEvaluator = (src, endowments, options = {}) => {
+    // Our sources have been already rewritten, so allow the hidden identifiers.
+    options.allowHidden = true;
```

Comment:
Should probably clone the options first to avoid side effect on function arguments.
```js
options = { ...options, allowHidden: true };
```